ts/client/src/client.ts

```diff
@@ -668,28 +678,39 @@ export class MangoClient {
 
   public async getMangoAccount(
     mangoAccount: MangoAccount | PublicKey,
+    loadSerum3Oo = false,
   ): Promise<MangoAccount> {
     const mangoAccountPk =
       mangoAccount instanceof MangoAccount
         ? mangoAccount.publicKey
         : mangoAccount;
-    return MangoAccount.from(
+    const mangoAccount_ = MangoAccount.from(
       mangoAccountPk,
       await this.program.account.mangoAccount.fetch(mangoAccountPk),
     );
+    if (loadSerum3Oo) {
+      await mangoAccount_?.reloadSerum3OpenOrders(this);
+    }
+    return mangoAccount_;
   }
 
   public async getMangoAccountForPublicKey(
```

Comment:
what's the difference between getMangoAccount and getMangoAccountForPublicKey -- doesn't getMangoAccount cover the latter function?